### PR TITLE
Add 2019 cfp details

### PR DESCRIPTION
**Pull request by dcwalk**

### CODE REVIEW

File: _includes/openhour.html [hunk 1]
```diff
@@ -10,11 +10,12 @@
...
   <div class="event-time-location">
     <div class="event-meta">
-      {% include icons/calendar-o.svg %}
-      <p class="event-meta-item">{{ call.date | date: "%-d %B %Y" }}, <strong>{{ call.startTime | date:"%-I:%M%P" }} – {{call.endTime | date:"%-I:%M%P"}}</strong></p></div>
-    <button download="call.ics" class="compact event-add-button" data-location="{{ event_location }}" data-summary="{{ event_title }}" data-description="{{ event_description }}" data-dtstart="{{ event_start }}" data-dtend="{{ event_end }}">{% include icons/calendar-plus-o.svg %}&nbsp;&nbsp;Add to Calendar (.ics)</button>
-    <a class="button compact" target="_blank" rel="noopener" href="https://www.google.com/calendar/render?action=TEMPLATE&{{ googleCalendarLink }}">
-      {% include icons/google_calendar.svg %}&nbsp;&nbsp;Add to Google Calendar
-    </a>
+      <p class="event-meta-item">{{ call.date | date: "%-d %B %Y" }}, <strong>{{ call.startTime | date:"%-I:%M%P" }} – {{call.endTime | date:"%-I:%M%P"}} EDT</strong></p>
```
Comment (dcwalk) [inline _includes/openhour.html]: I was thinking a bunch about how to do timezones... would writing some logic to convert/store the "primary" timezone in the data make sense? 

I also have this terrible preference about "EDT" but (go for "ET" instead) but no block on it :grimacing:

Comment (garrying) [reply _includes/openhour.html]: Yeah, that makes sense to me. Could be a follow-up PR

File: index.md [hunk 2]
```diff
@@ -8,15 +8,60 @@ redirect_from:
...
 
 {% include 2019-quotes.html %}
 
-## Updates
+We are inviting proposals for talks, workshops, discussions, demonstrations, art pieces, and interventions to explore this theme. Topics could include, but are in no way limited to:
+
+- Practices of reparative networking
+- Remediation practices for communication infrastructures
+- Uncooptable computing reconfigurations
+- New relations while refusing bad ones
```
Comment (ana0) [inline index.md]: Not sure about "New relations while refusing bad ones" maybe "bad ones" seems a bit vague? Otherwise all LGTM

**Review by ana0 — APPROVED** [review]

File: index.md (hunk 2 above)
Comment (benhylau) [reply index.md]: unfair, exploitative, unsustainable, asymmetric, unwelcomed

File: index.md [hunk 3]
```diff
@@ -8,15 +8,60 @@ redirect_from:
...
+[Submit Your Proposal](https://github.com/ournetworks/2019-submissions/issues/new){:.button.button-primary}
+
+<small>_Please note: first time GitHub users will need to [create a free account](https://github.com/join) before they can [open an issue](https://help.github.com/articles/creating-an-issue/)._</small>
+
+### Who Can Submit?
 
-Want to receive an email once the 2019 call for proposals is released?
+Anyone interested in how we can build better online futures!
+
+Our Networks is excited for sessions that centre the voices of those less often heard at technical events. We encourage BIPOC and LGBTQ2S folks to lead sessions and acknowledge the multiple marginalizations those with intersecting identities face. For 2019 we are also seeking to support those voices ~~in their preferred language~~ if you are interested in leading a session in a language other than English please indicate so in your submission.
```
Comment (benhylau) [inline index.md]: >~~in their preferred language~~

This crossed out is confusing.

**Review by benhylau — CHANGES_REQUESTED** [review]
Minor comment re: the crossed out text reading very awkward, unsure if intentional. Otherwise LGTM.

File: index.md (hunk 2 above)
Comment (dcwalk) [reply index.md]: Reconstructing technology beyond refusal

Comment (dcwalk) [reply index.md]: Reconstructing technology after refusal

Comment (benhylau) [inline index.md]:
```suggestion
- Reconstructing technology after refusal
```

Comment (dcwalk) [reply index.md]: Technology relations after refusal

Comment (dcwalk) [inline index.md]:
```suggestion
- Reconstructing technology after refusal
```

File: index.md (hunk 3 above)
Comment (dcwalk) [inline index.md]:
```suggestion
Our Networks is excited for sessions that centre the voices of those less often heard at technical events. We encourage BIPOC and LGBTQ2S folks to lead sessions and acknowledge the multiple marginalizations those with intersecting identities face. For 2019 we are also seeking to support those voices in their preferred language if you are interested in leading a session in a language other than English please indicate so in your submission.
```

File: index.md [hunk 4]
```diff
@@ -8,15 +8,60 @@ redirect_from:
...
```
Comment (benhylau) [inline index.md]:
```suggestion
Our Networks is excited for sessions that centre the voices of those less often heard at technical events. We encourage BIPOC and LGBTQ2S folks to lead sessions and acknowledge the multiple marginalizations those with intersecting identities face. For 2019 we are also seeking to support people in their preferred language. If you are interested in leading a session in a language other than English please indicate so in your submission.
```

**Review by benhylau — APPROVED** [review]

**Pull request merged**